### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Create Authentication Files" dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1388]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to table and observe the control type using inspect.

**Actual Result:**
Control type for table is list.
Narrator table commands such as "Caps+F3", "Caps+F8" are working fine.

**Narrator :**
DevDiv Key Vault, <say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">3</say-as>, selected, 

**NVDA:**
Select the subscriptions to create credentials for:  list
DevDiv Key Vault  check box  selected  not checked  Subscription ID: bd62906c-0a81-43c3-a2f8-126e4cf66ada

**Expected Result:**
Control type should be Table.
Narrator and NVDA should announce as
Table
DevDiv Key Vault, <say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">3</say-as>, selected, 

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![create authentication file-table-not defined](https://user-images.githubusercontent.com/22956130/35682268-1457b3be-0786-11e8-82d1-a000dae15fb2.jpg)
![create authentication file-table-not defined-nvda](https://user-images.githubusercontent.com/22956130/35682270-16fedce6-0786-11e8-9640-8ada47b4284c.jpg)

**Comment by Eskibear** [comment]
duplicated with #1405

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749849
